addons/godot-firebase/database/database.gd

```diff
@@ -1,10 +1,11 @@
 # ---------------------------------------------------- #
-#                 SCRIPT VERSION = 2.1                 #
+#                 SCRIPT VERSION = 2.2                 #
 #                 ====================                 #
 # please, remember to increment the version to +0.1    #
 # if you are going to make changes that will commited  #
 # ---------------------------------------------------- #
 
+tool
```

Comment:
Was there a specific reason to make these tool scripts? It occurred to me that nothing actually uses the tool functionality, although I could be completely wrong.

Reply:
Yep, these scripts don't work if they are used as EditorPlugins (which I'm using to make the FirestoreModeler, but in the future could be used for other Editor plugins or user's projects)